rio_cogeo/scripts/cli.py

```diff
@@ -258,24 +262,33 @@ def create(
         output_profile["blockxsize"] = blocksize
         output_profile["blockysize"] = blocksize
 
-    if web_optimized:
-        overview_blocksize = blocksize or 512
-
     config.update(
         {
             "GDAL_NUM_THREADS": threads,
             "GDAL_TIFF_INTERNAL_MASK": os.environ.get("GDAL_TIFF_INTERNAL_MASK", True),
-            "GDAL_TIFF_OVR_BLOCKSIZE": str(overview_blocksize),
         }
     )
 
+    tilematrixset: TileMatrixSet | None
```

Comment:
The `|` type hint will only work in python >10, I would prefer if we can't keep compatibility with lower version

```
tilematrixset: Optional[TileMatrixSet]
```